source/utilities.tex

```diff
@@ -12639,6 +12644,30 @@
 Subclause \ref{func.wrap} describes polymorphic wrapper classes that
 encapsulate arbitrary callable objects.
 
+\pnum
+Let \tcode{t} be an object of a type that is a specialization of
+\tcode{function}, \tcode{copyable_function}, or \tcode{move_only_function},
+such that the target object \tcode{x} of \tcode{t} has a type that
+is a specialization of
+\tcode{function}, \tcode{copyable_function}, or \tcode{move_only_function}.
```

Comment:
Does `function_ref` need to be mentioned here (@CaseyCarter, @MFHava)?

Reply:
As the intention was that there should be no double wrapping for "polymorphic function wrappers", I think `function_ref` should be mentioned here too.

Reply:
Such a change feels non-editorial to me.

Reply:
Should we open an LWG issue?